packages/core/core.ts

```diff
@@ -19,6 +19,9 @@ import * as fuzzer from "@jazzer.js/fuzzer";
 import * as hooking from "@jazzer.js/hooking";
 import { registerInstrumentor } from "@jazzer.js/instrumentor";
 import { trackedHooks } from "@jazzer.js/hooking";
+import * as process from "process";
+import * as tmp from "tmp";
```

Comment:
Use `tmp.setGracefulCleanup();`to cleanup tmp files on exit? See docs at https://github.com/raszi/node-tmp#graceful-cleanup